bioimageio/spec/utils/build_spec.py

```diff
@@ -0,0 +1,246 @@
+import os
+import datetime
+import hashlib
+from typing import Dict, List, Optional, Union
```

Comment:
```suggestion
from typing import Any, Dict, List, Optional, Union
```

if type hint suggestion below is accepted...